worlds/timespinner/LogicExtensions.py

```diff
@@ -92,7 +93,10 @@ def can_break_walls(self, state: CollectionState) -> bool:
             return True
 
     def can_kill_all_3_bosses(self, state: CollectionState) -> bool:
-        return state.has_all({'Killed Maw', 'Killed Twins', 'Killed Aelana'}, self.player)
+        if self.flag_prism_break:
+            return state.has_all({'Laser Access M', 'Laser Access I', 'Laser Access A'}, self.player)
+        else:
+            return state.has_all({'Killed Maw', 'Killed Twins', 'Killed Aelana'}, self.player)
```

Comment:
You can do this instead
```suggestion
        if self.flag_prism_break:
            return state.has_all({'Laser Access M', 'Laser Access I', 'Laser Access A'}, self.player)
        return state.has_all({'Killed Maw', 'Killed Twins', 'Killed Aelana'}, self.player)
```